node/src/components/storage.rs

```diff
@@ -104,8 +104,13 @@ const OS_FLAGS: EnvironmentFlags = EnvironmentFlags::WRITE_MAP;
 /// Mac OS X exhibits performance regressions when `WRITE_MAP` is used.
 #[cfg(target_os = "macos")]
 const OS_FLAGS: EnvironmentFlags = EnvironmentFlags::empty();
+use static_assertions::const_assert;
+use std::mem;
+const _EVENT_SIZE: usize = mem::size_of::<Event>();
```

Comment:
Are we cleaning this up later? We actually want an event size restriction on the top level event, right?

If so, a component's event should probably be 95 (`repr(u8)`) or 88 bytes at most, to allow for an `enum` variant tag of the inevitable top level reactor event.

Reply:
Added Event Specific limits for all those events which have been re-factored